app/models/submission.rb

```diff
@@ -144,6 +144,16 @@ def cache_access_level
   scope :final_is_restricted_institution, -> { where('status LIKE "released for publication%"').where(access_level: 'restricted_to_institution') }
   scope :final_is_withheld, -> { where('status LIKE "released for publication%"').where(access_level: 'restricted') }
   scope :ok_to_release, -> { where('released_for_publication_at <= ?', Time.zone.today.end_of_day) }
+  scope :ok_to_autorelease, -> {
+                              ok_to_release.where('released_metadata_at <= ?', Time.zone.today.years_ago(2).end_of_day)
```

Comment:
Should this be `'released_for_publication_at <= ?', Time.zone.today` ?  If something gets extended, it's `released_metadata_at` will be more than two years ago.

Reply:
Yeah you're right. I think when I was writing that, I thought released_for_publication wasn't set until it was actually released